hermes/views.py

```diff
@@ -237,21 +290,22 @@ def get(self, request, *args, **kwargs):
         
         {title: <Title of the message>,
          topic: <kafka topic to post message to>, 
-         author: <submitter of the message>,
+         submitter: <submitter of the message>,
```

Comment:
What's the use-case for author and submitter being separate here?

Reply:
Author would be the people (possibly a long list) who are responsible for crafting the message, the submitter is the username/email/TOM the report came from. So, for Neoexchange, author might be `Tim, L.; Chatelain, J.; Gomez, E.` and submitter would be `neox@lco.global`.

Reply:
I see, OK.